src/main/java/io/javalin/http/JavalinResponseWrapper.kt

```diff
@@ -26,23 +29,41 @@ class ResponseWrapperContext(request: HttpServletRequest, val config: JavalinCon
 
 class OutputStreamWrapper(val res: HttpServletResponse, val rwc: ResponseWrapperContext) : ServletOutputStream() {
 
+    val streamBuffer = ByteArrayOutputStream()
+    var sizeLimitExceeded = false
+
+    var gzipCompressor: MyGzipOutputStream? = null
```

Comment:
Is the plan to add a brotliCompressor later? Would it make sense to make this `compressorOutputStream: OutputStream`, then give it either a `GzipOutputStream` or `BrotliOutputStream` later?

Reply:
Yes, that's the plan. Good call